environment.yml

```diff
@@ -0,0 +1,13 @@
+# navigate to the directory which contains this file and run `conda env create -f environment.yml` to build
+name: bp
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - bicycleparameters
```

Comment:
For development you wouldn't want to install bicycleparameters. I do one of two things 1) cd into the bicycleparameters directory and work from there or 2) use `python setup.py develop` so that it is installed from the local directory.

Reply:
True, if you installed it then you wouldn't be using your local developing version. I'll remove.